core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -2057,11 +2066,15 @@ private[spark] class DAGScheduler(
    *   reprocessing for follow-on fetch failures)
    */
   private def removeExecutorAndUnregisterOutputs(
-      execId: String,
+      execIdOpt: Option[String],
       fileLost: Boolean,
       hostToUnregisterOutputs: Option[String],
       maybeEpoch: Option[Long] = None,
       ignoreShuffleFileLostEpoch: Boolean = false): Unit = {
+    // This method assumes an executor lost. Thus, it should be no-op for
+    // the storage system whose location isn't based on executors.
```

Comment:
Could you revise the function description accordingly?

Reply:
Sure.